static/js/app.js

```diff
@@ -682,3 +682,18 @@ $(document).ready(function() {
 $(document).ready(function() {
     $('.alert').createAlert();
 });
+
+/* ==========================================================================
+   Footer Button: Scroll to Top
+   ========================================================================== */
+
+$(document).ready(function() {
```

Comment:
I notice we're using both in app.js. It probably makes sense to standardize them all at some point in the front-end standards. I know I prefer the long-hand version, but I'm not picky.